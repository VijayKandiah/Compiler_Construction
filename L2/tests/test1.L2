(:go
  (:go
    0 0
    %v1 <- +5
    return
  )
)
