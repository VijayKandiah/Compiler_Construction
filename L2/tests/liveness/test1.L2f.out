(
(in
(r12 r13 r14 r15 rax rbp rbx)
(myVar1 r12 r13 r14 r15 rax rbp rbx)
(myVar1 myVar2 r12 r13 r14 r15 rax rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
)

(out
(myVar1 r12 r13 r14 r15 rax rbp rbx)
(myVar1 myVar2 r12 r13 r14 r15 rax rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
()
)

)
